packages/blockchain-link-utils/package.json

```diff
@@ -17,6 +17,8 @@
         "prepublish": "yarn tsx ../../scripts/prepublish.js"
     },
     "dependencies": {
+        "@mobily/ts-belt": "^3.13.1",
```

Comment:
this is the first usage of @mobility/ts-belt on this layer. ok for feat/solana but we might revisit this later